### ISSUE TITLE
Is it possible to use a TextureView as a Renderable?

### ISSUE THREAD

**Issue by vicmns** [google-ar/arcore-android-sdk#522]
So I've been playing with the ViewRenderable class, which pretty much can render any View that uses the common APIs for rendering (which is awesome BTW).

Now, I have a custom view that extends from a TextureView, in which I use C++/OpenGL to do the rendering. Doing a Quick test using this view inside a ViewRendereable it wont render (which makes since since ARSceneView it's a SurfaceView). So I have two questions here:

1. It is possible to use a TextureView as a custom renderable in ARCore?
2. If not, is there a way to have a delimited area (anchored in a given point) where I can freely make my JNI/OpenGL calls?

Basically what I'm trying to achieve is (in the case that the TextureView can't be used inside a rendereable) to have an area to work as a Plane where I can freely use my C++ calls in order to do the render.

I already asked this question in [SO](https://stackoverflow.com/questions/51718234/android-arcore-use-textureview-in-a-viewrenderable), but I think this is a better place to ask help on this...

**Comment by inio** [comment]
Please post issues related to Sceneform on the [google-ar/sceneform-android-sdk issues board](https://github.com/google-ar/sceneform-android-sdk/issues).
